modules/client/examples.md

```diff
@@ -1093,7 +1093,7 @@ console.log(proposals);
 
 ```ts
 import {
-  ClientAddressList,
+  AddresslistVotingClient,
```

Comment:
Really small... but worth it?

```suggestion
  AddressListVotingClient,
```

Reply:
Nope, the SDK used `AddressList` and the contracts `Addresslist` this pr also solves this, all the `addressList` and `AdressList` are `addresslist` and `AddressList` respectively